static/app/views/issueDetails/groupDetails.spec.tsx

```diff
@@ -49,29 +43,33 @@ describe('groupDetails', () => {
     routes,
   };
 
-  const defaultInit = initializeOrg({
+  const defaultInit = initializeOrg<{groupId: string}>({
     project,
     router: initRouter,
   });
 
-  function MockComponent({group: groupProp, environments, eventError}) {
+  function MockComponent({
+    group: groupProp,
+    environments,
+    eventError,
+  }: {
+    environments?: Environment[];
+    eventError?: boolean;
+    group?: Group;
+  }) {
     return (
       <div>
         Group Details Mock
-        <div>title: {groupProp.title}</div>
-        <div>environment: {environments.join(' ')}</div>
+        <div>title: {groupProp?.title}</div>
+        <div>environment: {environments?.join(' ')}</div>
         {eventError && <div>eventError</div>}
       </div>
     );
   }
 
   const createWrapper = (init = defaultInit) => {
     return render(
-      <GroupDetails
-        {...init.router}
-        router={init.router}
-        organization={init.organization}
-      >
+      <GroupDetails {...init.router} {...init.routerProps} router={init.router}>
```

Comment:
Do you have to spread `init.router? doesn't routerProps include that stuff? also do you need `router={init.router}` doesn't routerProps also include that?

Reply:
Yeah it doesn't need those, removed